codex-rs/core/src/codex.rs

```diff
@@ -540,7 +541,9 @@ impl Session {
         // TODO: Those items shouldn't be "user messages" IMO. Maybe developer messages.
         let mut conversation_items = Vec::<ResponseItem>::with_capacity(2);
         if let Some(user_instructions) = turn_context.user_instructions.as_deref() {
-            conversation_items.push(Prompt::format_user_instructions_message(user_instructions));
+            conversation_items.push(ResponseItem::from(UserInstructions::new(
+                user_instructions.to_string(),
+            )));
```

Comment:
This might be more concise?

```suggestion
            conversation_items.push(UserInstructions::new(user_instructions.to_string()).into());
```